README.md

```diff
@@ -1,424 +1,159 @@
+* [Homepage](https://github.com/googleapis/google-cloud-ruby/)
+* [Reference documentation](https://cloud.google.com/ruby/docs/reference)
 
-- `gem install google-cloud-os_login`
-- [google-cloud-os_login README](google-cloud-os_login/README.md)
-- [google-cloud-os_login API documentation](https://googleapis.dev/ruby/google-cloud-os_login/latest)
-- [google-cloud-os_login on RubyGems](https://rubygems.org/gems/google-cloud-os_login)
-- [Google Cloud DNS documentation](https://cloud.google.com/compute/docs/oslogin/rest/)
+This repository includes client libraries for Google Cloud Platform services,
+along with a selected set of Google services unrelated to the cloud platform.
 
-### Phishing Protection
+Most libraries here are automatically generated by the
+[GAPIC Generator](https://github.com/googleapis/gapic-generator-ruby). A small
+number are written and maintained by hand.
 
-- `gem install google-cloud-phishing_protection`
-- [google-cloud-phishing_protection README](google-cloud-phishing_protection/README.md)
-- [google-cloud-phishing_protection API documentation](https://googleapis.dev/ruby/google-cloud-phishing_protection/latest)
-- [google-cloud-phishing_protection on RubyGems](https://rubygems.org/gems/google-cloud-phishing_protection)
-- [Phishing Protection documentation](https://cloud.google.com/phishing-protection/docs/)
+## Using the client libraries
+
+These client library rubygems each include classes and methods that can be used
+to make authenticated calls to specific Google APIs. Some libraries also
+include additional convenience code implementing common client-side workflows
+or best practices.
 
-### Cloud Pub/Sub
+In general, you can expect to:
 
-- `gem install google-cloud-pubsub`
-- [google-cloud-pubsub README](google-cloud-pubsub/README.md)
-- [google-cloud-pubsub API documentation](https://googleapis.dev/ruby/google-cloud-pubsub/latest)
-- [google-cloud-pubsub on RubyGems](https://rubygems.org/gems/google-cloud-pubsub)
-- [Google Cloud Pub/Sub documentation](https://cloud.google.com/pubsub/docs)
+1.  Activate access to the API by creating a project on the Google Cloud
+    Console, enabling billing if necessary, and enabling the API.
 
-### Recaptcha Enterprise
+2.  Install a library, usually by adding it to your bundle. For example, here
+    is how you might add a the Translation service client to your Gemfile:
 
-- `gem install google-cloud-recaptcha_enterprise`
-- [google-cloud-recaptcha_enterprise README](google-cloud-recaptcha_enterprise/README.md)
-- [google-cloud-recaptcha_enterprise API documentation](https://googleapis.dev/ruby/google-cloud-recaptcha_enterprise/latest)
-- [google-cloud-recaptcha_enterprise on RubyGems](https://rubygems.org/gems/google-cloud-recaptcha_enterprise)
-- [Recaptcha Enterprise documentation](https://cloud.google.com/recaptcha-enterprise/docs/)
+    ```ruby
+    # Gemfile
 
-### Recommender
+    # ... previous libraries ...
+    gem "google-cloud-translate", "~> 3.2"
+    ```
 
```

Comment:
should we mention a reference to authentication as a step ? or add it to client object creation step ?

Reply:
Yeah, good call, I'll add a note to the object creation step.

Reply:
Done